README.md

```diff
@@ -1,4 +1,3 @@
-
 # HTTP Request/Response Cycle - Lab
 
 ## Introduction 
```

Comment:
The PR interface isn't letting me comment directly on the lines that _didn't_ change, but I think we should change a couple more places:
1. [Here](https://github.com/learn-co-curriculum/dsc-http-request-response-lab/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R22), lines 22-24 under the `## Open Notify` heading, let's reorder the bullet points so that overhead pass predictions is third
2. [Here](https://github.com/learn-co-curriculum/dsc-http-request-response-lab/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R36), lines 36-37 under the `### API Endpoints` heading, let's reorder the list of endpoints so that overhead pass predictions is third
3. [Here](https://github.com/learn-co-curriculum/dsc-http-request-response-lab/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R33), line 33 under `### API Endpoints` it says "we have following end points". Let's change that to "we have the following endpoints" (adding "the", removing the space between "end" and "points")